src/Modules/ContentBlocks/templates/Frontend/Widgets/Detail.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if content_block is defined and content_block.text %}
+  <div class="module-content-blocks widget-content-blocks-default content-block-{{ content_block.id }}" id="{{ content_block.id }}">
```

Comment:
`id="{{ content_block.id }}"` will generate an numeric string. `id` can't start with a number. Shouldn't this be `data-id`? I think `data-role="content-block" data-id="{{ content_block.id }}"` would be more logical.

Also: why the classes?